docs/security.md

```diff
@@ -1,5 +1,32 @@
 # Security
 
+Start by reviewing the [OpenTelemetry Collector security
+documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security.md).
+
+## Reporting Security Issues
+
+Please *DO NOT* report security vulnerabilities via public GitHub issue
+reports. Please [report security issues here](
+https://www.splunk.com/en_us/product-security/report.html).
+
+## Dependencies
+
+This project relies on a variety of [external
+dependencies](https://github.com/signalfx/splunk-otel-collector/network/dependencies).
+These dependencies are monitored by
+[Dependabot](https://docs.github.com/en/code-security/supply-chain-security/configuring-dependabot-security-updates).
+Dependencies are [checked
+daily](https://github.com/signalfx/splunk-otel-collector/blob/main/.github/dependabot.yml)
+and associated pull requests are opened automatically. Upgrading to the [latest
+release](https://github.com/signalfx/splunk-otel-collector/releases)
+is recommended to ensure you have the latest security updates. If a security
+vulnerability is detected for a dependency of this project then either:
+
+- You are running an older release
+- A new release with the updates has not been cut yet
+- The updated dependency has not been merged likely due to some breaking change
```

Comment:
Should we mention the process of resolving this to help set expectations?

Reply:
good call -- fixed